app/services/sso_service.rb

```diff
@@ -39,14 +39,6 @@ def persist_authentication!
     existing_user.destroy if new_login?
 
     if valid?
-      if new_login?
-        # FIXME: possibly revisit this. Is there a possibility that different sign-in contexts could get
-        # merged? MHV LOA1 -> IDME LOA3 is ok, DS Logon LOA1 -> IDME LOA3 is ok, everything else is not.
-        # because user, session, user_identity all have the same TTL, this is probably not a problem.
-        MERGABLE_IDENTITY_ATTRIBUTES.each do |attribute|
```

Comment:
Since you removed this, it looks like you can delete the `MERGABLE_IDENTITY_ATTRIBUTES` declaration above too.

Reply:
yup, good catch.